### PR TITLE
AzureTestPlan: Changing multi select language to single select.

### PR DESCRIPTION
**Pull request by t-shreysingh**
**Task name**:  AzureTestPlan: Changing multi select language to single select.

**Description**: 

**Documentation changes required:** (Y/N) <Please mark if documentation changes are required>

**Added unit tests:** (Y/N) <Please mark if unit tests were added or updated according changes>

**Attached related issue:** (Y/N) <Please add link to related issue here>

**Checklist**:
- [ ] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected
image

### CODE REVIEW

File: Tasks/AzureTestPlanV0/task.json [hunk 1]
```diff
@@ -81,9 +81,6 @@
         "Java-Gradle": "Java-Gradle",
```
Comment (triptijain2112) [inline Tasks/AzureTestPlanV0/task.json:81]: please update task version also

Comment (t-shreysingh) [reply Tasks/AzureTestPlanV0/task.json:81]: Sure.

**Comment by t-shreysingh** [comment]
close it

**Pull request closed without merging**